js/controller.js

```diff
@@ -36,6 +36,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
       "adOverlayUrl": null,
       "showAdOverlay": false,
       "showAdOverlayCloseButton": false,
+      "adOverlayLoaded": false,
```

Comment:
Trying to think through the use case, do we actually need to store this state or just pass the message along the message bus? I'm not coming up with any reason the skin needs to care about the overlay being loaded or not aside from the check in adOverlay.js .